destination/db/clickhouse.go

```diff
@@ -73,7 +77,20 @@ func GetClickHouseConnection(configuration map[string]string) (*ClickHouseConnec
 		log.Error(err)
 		return nil, err
 	}
-	return &ClickHouseConnection{conn}, nil
+	err = retry.OnNetError(func() error {
+		return conn.Ping(ctx)
+	}, ctx, "ping", false)
+	if err != nil {
+		if err.Error() == "EOF" {
```

Comment:
This scares me :) is there any particular type we can check?

Reply:
no, it's just EOF.